app/components/Forms/AddCommentField.tsx

```diff
@@ -52,21 +58,23 @@ const ProblemReportField: React.FunctionComponent<FieldProps<string>> = (
       `}</style>
       <span className="title">
         <Button variant="link" onClick={() => setExpanded(true)}>
-          <FontAwesomeIcon icon={titleIcon} />
+          <FontAwesomeIcon icon={faCommentAlt} />
           &nbsp;
           {title}
         </Button>
       </span>
       {expanded && (
         <div className="report-field">
+          <legend>CIIP Application Comment</legend>
+
+          <p>
+            This area is only for leaving a comment regarding the data that you
+            reported on this page. This comment will be visible by the person
+            reviewing this CIIP application once the application is submitted.
+          </p>
           <p>
-            Please leave a detailed message below explaining any issues
-            encountered on this page.
-            <br />
-            This message will be sent with your application when it is
-            submitted.
-            <br />A representative from the Climate Action Secretariat may
-            contact you for further clarification.
+            For any technical issue, or to review documentation, please click on
+            the help bubble in the bottom right of your screen.
```

Comment:
Minor detail (what they'll see is a button):
```suggestion
            the help button in the bottom right of your screen.
```
I'm also wondering if we can delay adding this sentence and add it as part of the card implementing the help button (could cause confusion for QA testers)?

Reply:
I would expect that help button to be in our next release to QA

Reply:
OK. Either way, "button" is more correct - they will not see any bubble in the bottom of the screen.